### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by Dipanita45**
Added security policy in the website to protect it against harms

#109 

Please merge my pull request and label me level2
@c0sm0void

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Security Policy
+Supported Versions
+The following versions of Revot are currently supported with security updates:
+
+Version	Supported
```
Comment (whyvineet) [inline SECURITY.md:5]: Add proper Python version and dependencies version

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+1.x.x	✅ Supported
+0.x.x	❌ Not supported
+Reporting a Vulnerability
+If you discover a security vulnerability in Revot, we encourage you to report it as soon as possible. We will investigate all legitimate reports and do our best to quickly fix the issue.
+
+# How to Report
+Please report vulnerabilities by emailing us at revot@gmail.com. Include as much detail as possible to help us identify and fix the issue swiftly.
```
Comment (whyvineet) [inline SECURITY.md:12]: Check the email from the README.md and change this also!

**Review by whyvineet — CHANGES_REQUESTED** [review]
Give the proper description of dependency versions and enhance the SECURITY.md by making the changes mentioned in the review!

**Comment by Dipanita45** [comment]
Please can you give me your Gmail ID

**Comment by Dipanita45** [comment]
I can't find it

**Comment by whyvineet** [comment]
**Email** - [minhajulhossain@gmail.com](mailto:minhajulhossain@gmail.com)

@Dipanita45

**Comment by Dipanita45** [comment]
Please review and merge it and label it
@c0sm0void

**Pull request merged**